pymess/migrations/0030_migration.py

```diff
@@ -0,0 +1,310 @@
+# Generated by Django 4.2.13 on 2024-07-03 18:01
+
+import django.db.models.deletion
+from django.db import migrations, models
+from utils.migrations import migration_print
```

Comment:
I'll remove these prints before merging.

Reply:
I think you can do it.